app/models/like.rb

```diff
@@ -1,2 +1,17 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] Here also, Like belongs to a user with a column name of `author_id` instead of the `user_id`. please apply the `belongs_to :author, class_name: 'User'` to preserve this behavior.